pv_site_api/main.py

```diff
@@ -94,6 +94,9 @@ def traces_sampler(sampling_context):
     allow_headers=["*"],
 )
 
+# Uncomment once environment variables are set for enode
+# enode_client = httpx.Client(auth=EnodeAuth())
+
```

Comment:
I think we'll ultimately want to use a [context-based client](https://www.python-httpx.org/advanced/#:~:text=a%20Client%20instance.-,Usage,-The%20recommended%20way) instead of a global variable, so these lines can be removed.